compose/foundation/foundation/build.gradle

```diff
@@ -84,6 +84,15 @@ if(AndroidXComposePlugin.isMultiplatformEnabled(project)) {
     }
 
     kotlin {
+        macosX64 {
+            binaries.findTest(NativeBuildType.DEBUG).freeCompilerArgs +=
+                    ["-linker-option", "-framework", "-linker-option", "Metal"]
+        }
+        macosArm64 {
+            binaries.findTest(NativeBuildType.DEBUG).freeCompilerArgs +=
+                    ["-linker-option", "-framework", "-linker-option", "Metal"]
+        }
+
```

Comment:
It is not required here. Already applied in another graphics modules (like compose:ui:ui-graphics)
```suggestion
```

Reply:
Ok. I understand.
Then my suggest is not correct and you can skip it.